libcloud/security.py

```diff
@@ -61,6 +65,19 @@
     '/etc/ssl/certs/YaST-CA.pem',
 ]
 
+# Insert certifi CA bundle path to the front of Libcloud CA bundle search
+# path if certifi is available
+try:
+    import certifi
+except ImportError:
+    has_certifi = False
+else:
+    has_certifi = True
+
+if has_certifi and USE_CERTIFI:
+    certifi_ca_bundle_path = certifi.where()
+    CA_CERTS_PATH.insert(0, certifi_ca_bundle_path)
```

Comment:
Defaulting it to False would require user interaction which is not ideal, but then again Python packaging / virtual environment world is far from ideal (ideally, every app, etc. would always run in an isolated virtual environment, but in real life that's not always the case) so making it explicit and defaulting to False might indeed be a better choice (this way it will be an explicit choice and people won't be surprised).